src/test/url-handling.test.js

```diff
@@ -109,10 +111,8 @@ function _getStoreWithURL(
 }
 
 // Serialize the URL of the current state, and create a new store from that URL.
-function _getStoreFromStateAfterUrlRoundtrip(
-  state: State,
-  profile: Profile | null = getProfile()
-): Store {
+function _getStoreFromStateAfterUrlRoundtrip(state: State): Store {
+  const profile = getProfile(state);
```

Comment:
In your previous version, this wasn't taking the profile from the state, I believe this was a mistake.